bindings/python/FloatingBaseEstimators/src/FloatingBaseEstimators.cpp

```diff
@@ -38,7 +38,24 @@ void CreateKinDynComputationsDescriptor(pybind11::module& module)
     py::class_<KinDynComputationsDescriptor>(module, "KinDynComputationsDescriptor")
         .def(py::init())
         .def_readwrite("kindyn", &KinDynComputationsDescriptor::kindyn)
-        .def("is_valid", &KinDynComputationsDescriptor::isValid);
+        .def("is_valid", &KinDynComputationsDescriptor::isValid)
+        .def("get_nr_of_dofs", &KinDynComputationsDescriptor::getNrOfDegreesOfFreedom)
+        .def("get_joint_pos", &KinDynComputationsDescriptor::getJointPos,
+              py::arg("q"))
```

Comment:
```suggestion
              py::arg("s"))
```

Reply:
https://github.com/dic-iit/bipedal-locomotion-framework/pull/303#discussion_r631066529

Reply:
Argument not required anymore after https://github.com/dic-iit/bipedal-locomotion-framework/pull/303/commits/3d3653036c2a534ff2126f9bf924755c991ccb13